cli/src/pcluster/models/imagebuilder.py

```diff
@@ -78,20 +83,62 @@ def __init__(self, message: str, validation_failures: list = None):
         self.validation_failures = validation_failures or []
 
 
+class LimitExceededImageBuilderActionError(ImageBuilderActionError):
+    """Represent an error during the execution of an action on the imagebuilder due to limit exceeded."""
+
+    def __init__(self, message: str):
+        super().__init__(message)
+
+
+class BadRequestImageBuilderActionError(ImageBuilderActionError):
```

Comment:
Is BadRequestImageBuilderActionError mapping to BadRequestException and is ImageBuilderActionErrormapping to InternalServiceException? What are different use cases for BadRequestException and InternalServiceException?

Reply:
The `BadRequestException` represents an exception due to some problem in the request (for example, the customer attempts to delete an image that is shared without using the `force` option), it's a 4xx. whereas an `InternalServiceException` is a generic Server side exception (a 500). In the case of `AWSClientError` returned by AWS calls, I will separate these cases, as with @demartinofra we determined that, for example, `ParameterValidationError`s  should be surfaced as `BadRequestException`s